packages/cli/package.json

```diff
@@ -72,8 +71,9 @@
     "esbuild": "^0.25.1",
     "glob": "^11.0.0",
     "micromatch": "^4.0.7",
+    "nanospinner": "^1.2.2",
```

Comment:
I'd rather use `ora` than `nanospinner` which has far fewer downloads, stars, and contributors. `picocolors` looks good.